service/src/main/kotlin/no/nav/su/se/bakover/service/behandling/BehandlingServiceImpl.kt

```diff
@@ -0,0 +1,175 @@
+package no.nav.su.se.bakover.service.behandling
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import no.nav.su.se.bakover.database.behandling.BehandlingRepo
+import no.nav.su.se.bakover.database.beregning.BeregningRepo
+import no.nav.su.se.bakover.database.hendelseslogg.HendelsesloggRepo
+import no.nav.su.se.bakover.database.oppdrag.OppdragRepo
+import no.nav.su.se.bakover.domain.AktørId
+import no.nav.su.se.bakover.domain.Attestant
+import no.nav.su.se.bakover.domain.Behandling
+import no.nav.su.se.bakover.domain.Saksbehandler
+import no.nav.su.se.bakover.domain.behandling.Behandlingsinformasjon
+import no.nav.su.se.bakover.domain.beregning.Fradrag
+import no.nav.su.se.bakover.domain.oppdrag.NyUtbetaling
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimuleringClient
+import no.nav.su.se.bakover.domain.oppdrag.simulering.SimuleringFeilet
+import no.nav.su.se.bakover.domain.oppdrag.utbetaling.UtbetalingPublisher
+import no.nav.su.se.bakover.domain.oppgave.KunneIkkeOppretteOppgave
+import no.nav.su.se.bakover.domain.oppgave.OppgaveClient
+import no.nav.su.se.bakover.domain.oppgave.OppgaveConfig
+import no.nav.su.se.bakover.service.sak.SakService
+import no.nav.su.se.bakover.service.søknad.FantIkkeSøknad
+import no.nav.su.se.bakover.service.søknad.SøknadService
+import no.nav.su.se.bakover.service.utbetaling.UtbetalingService
+import java.time.LocalDate
+import java.util.UUID
+
+internal class BehandlingServiceImpl(
+    private val behandlingRepo: BehandlingRepo,
+    private val hendelsesloggRepo: HendelsesloggRepo,
+    private val beregningRepo: BeregningRepo,
+    private val oppdragRepo: OppdragRepo,
+    private val simuleringClient: SimuleringClient,
+    private val utbetalingService: UtbetalingService, // TODO use services or repos? probably services
```

Comment:
Jeg synes det gir mening og bruke services, vi kan godt ha en simuleringService og oppdragService, fordi den service-koden blir nå duplisert. Jeg heller mot at det eneste repoet BehandlingService bør ha tilgang på er behandlingRepo

Reply:
Servicene slik de er nå er bare kastet sammen for å gi oss samme funksjonalitet som vi hadde. Naturlig at disse refaktoreres og splittes opp mindre servicer etterhvert.